<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoplayer</class>
 <widget class="QWidget" name="videoplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="playbutton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="media-preview.qrc">
     <normaloff>:/media-preview/play</normaloff>:/media-preview/play</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pausebutton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="media-preview.qrc">
     <normaloff>:/media-preview/pause</normaloff>:/media-preview/pause</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="stopbutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="media-preview.qrc">
     <normaloff>:/media-preview/stop</normaloff>:/media-preview/stop</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="Phonon::VolumeSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="Phonon::SeekSlider" name="seekSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="Phonon::VideoWidget" name="videoDisplay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="media-preview.qrc"/>
 </resources>
 <connections/>
</ui>
